data-default-class/data-default-class.cabal

```diff
@@ -1,6 +1,6 @@
 Name:            data-default-class
-Version:         0.1.0
-Cabal-Version:   >= 1.6
+Version:         0.1.0.1
```

Comment:
PS: if you feel better with 0.1.1, I'll happily update; a major version bump to 0.2 however is overkill